crypto/multisig/wire.go

```diff
@@ -0,0 +1,26 @@
+package multisig
+
+import (
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+)
+
+// TODO: Figure out API for others to either add their own pubkey types, or
+// to make verify / marshal accept a cdc.
+const (
+	ThresholdPubkeyAminoRoute = "tendermint/ThresholdMultisigPubkey"
```

Comment:
We use eg. `tendermint/PubKeyEd25519` so this should be `tendermint/PubKeyMultisigThreshold`